draft-ietf-tls-tls13.md

```diff
@@ -2247,8 +2277,8 @@ extensions
 
 When this message will be sent:
 
-> A non-anonymous server can optionally request a certificate from the client,
-if appropriate for the selected cipher suite. This message, if sent, will
+> A server which is authenticating with a certificate can optionally
```

Comment:
There's another mention of anonymous at the end of the extension that probably should be correspondingly tweaked.